com.unity.formats.fbx/Editor/FbxExportSettings.cs.meta

```diff
@@ -1,11 +0,0 @@
-fileFormatVersion: 2
-guid: 03d96513d037ea843ab436321bab4c33
-MonoImporter:
```

Comment:
can you please rename the meta file as well? Could give warnings about missing meta file if we do not include it.